src/app/pages/news/12.0/03-core.md

```diff
@@ -1,12 +1,12 @@
 ## Core {#core}
 
-We not only updated the Designer but also the Axon Ivy Core got some great new features:
+The Axon Ivy Core also has some great new features:
 
-- **Notification**: Task and business notifications are delivered over different notification channels (Displayed in Portal, sent via mail, or Microsoft Teams). Send your own business notifications over a new API. Users can configure which notifications they want to receive over which notification channels.
-- **CRON**: You can now configure periodical jobs, start and intermediate events by using CRON expressions. CRON expressions give you a broad spectrum of possible configurations. From once every minute to every day, week, month, or year.
-- **Developer Workflow UI**: Improved developer workflow UI with a lot of new features like task and case notes, simple statistics, notifications, and improved task, case, and process start lists.
-- **S3 Document Storage**: Workflow documents can now be saved in S3-compatible storage. S3 has many advantages over the conventional local file system.
-- **Java 21**: You can now use the latest language features of Java 21 in your projects. Profit of all the new performance optimizations of Java 21 when running your Engine.
+- **Notification**: Task and business notifications can now be delivered across multiple channels, including the Portal, email, or Microsoft Teams. A new API has been introduced for sending custom business notifications, and users can configure which notifications they want to receive and through which channel.

```

Comment:
Notification can be received through multiple channels. So it should be: and through which channels.